bundles/org.openhab.binding.fenecon/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,177 @@
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="home-device">
+
+		<label>FENECON Home</label>
+		<description>Thing for FENECON Home Device</description>
+		<category>Solarplant</category>
+
+		<channels>
+			<channel id="state" typeId="state"/>
+			<channel id="last-update" typeId="last-update"/>
+			<channel id="ess-soc" typeId="ess-soc"/>
+			<channel id="charger-power" typeId="charger-power"/>
+			<channel id="discharger-power" typeId="discharger-power"/>
+			<channel id="emergency-power-mode" typeId="emergency-power-mode"/>
+			<channel id="consumption-active-power" typeId="consumption-active-power"/>
+			<channel id="consumption-active-power-l1" typeId="consumption-active-power-phase"/>
+			<channel id="consumption-active-power-l2" typeId="consumption-active-power-phase"/>
+			<channel id="consumption-active-power-l3" typeId="consumption-active-power-phase"/>
+			<channel id="consumption-max-active-power" typeId="consumption-max-active-power"/>
+			<channel id="production-max-active-power" typeId="production-max-active-power"/>
+			<channel id="production-active-power" typeId="production-active-power"/>
+			<channel id="export-to-grid-power" typeId="export-to-grid-power"/>
+			<channel id="exported-to-grid-energy" typeId="exported-to-grid-energy"/>
+			<channel id="import-from-grid-power" typeId="import-from-grid-power"/>
+			<channel id="imported-from-grid-energy" typeId="imported-from-grid-energy"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the FENECON device, e.g. 192.168.1.11</description>
+			</parameter>
+			<parameter name="password" type="text">
+				<context>password</context>
+				<label>Password</label>
+				<default>user</default>
+				<description>Password to access the device. The password for guest access is set by default.</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="port" type="integer" min="1">
+				<context>network-address</context>
+				<label>Port</label>
+				<description>Port of the FENECON device</description>
+				<default>8084</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="1">
+				<label>Refresh Interval</label>
+				<description>Interval the device is polled in sec.</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Channel Types -->
+	<channel-type id="state">
+		<item-type>String</item-type>
+		<label>System State</label><!-- Use uppercase words, except prepositions. 2-3 words, max 25 chars -->
```

Comment:
```suggestion
		<label>System State</label>
```

Reply:
I have removed the unnecessary tags from the pom.xml. However, the connection tag is included in the addon.xml. Or do you mean something else?